cmd/defaults.yaml

```diff
@@ -1,8 +1,22 @@
+# Log describes the output to the stdout.
 Log:
+  # Following log levels are allowed and print this and above levels.
+  # 1. panic:           Highest level of severity. After the log, ZITADEL will terminate with an error
+  # 2. fatal:           After this log ZITADEL will terminate with exit code 1
+  # 3. error:           Issues that definitely should be noted.
+  # 4. warn / warning:  Non-critical entries that deserve eyes. 
+  # 5. info / "":            General operational entries about what's going on inside the application.
+  # 6. debug:           Usually not intended for production. Very verbose logging.
+  # 7. trace:           Designates finer-grained informational events than the Debug.
   Level: debug
+  # Defines how the log data are formatted before sent to stdout
   Formatter:
+    # Allowed values are:
+    # text / "":  prints logs like `time="yyyy-mm-ddThh:mm:ssZ" level=logLevel msg="message"`
+    # json:       prints logs like `{"time": "yyyy-mm-ddThh:mm:ssZ", "level": "logLevel", "msg": "message"}`
     Format: text
 
+# The port ZITADEL listens for requests
 Port: 8080
 ExternalPort: 8080
```

Comment:
What do you think about my description here https://github.com/zitadel/zitadel/pull/3817#discussion_r895858122?